src/main/resources/config/application.yml

```
@@ -0,0 +1,17 @@
+spring:
+  config:
+    import: mail-templates.yml
+  mail:
+    host: host
+    port: 587
+    username: username
+    properties:
+      mail:
+        smtp:
+          auth: true
+          starttls:
+            enable: true
+            required: true
+activity:
+  root:
+    path: /DATA/
```

All of these property could change based on the deploy configuration, we should give the possibility to override this using environment variables, with ENV names explicitly defined

I'm reporting a suggestion just for the first as an example, but you should do for each property that could change (all of these should be configurable):
```suggestion
    host: ${MAIL_HOST:host}
    port: 587
    username: username
    properties:
      mail:
        smtp:
          auth: true
          starttls:
            enable: true
            required: true
activity:
  root:
    path: /DATA/
```